routes/task_progress.py

```
@@ -0,0 +1,376 @@
+"""
+Celeryタスクの進捗監視とキャンセル機能を提供するFlaskルート
+
+Server-Sent Events (SSE)を使用してリアルタイムでタスクの状態を配信し、
+部分レスポンスの表示やタスクキャンセル機能を提供する
+"""
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)